src/SDKs/Cdn/Cdn.Tests/Cdn.Tests.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Cdn.Tests</PackageId>
     <Description>Cdn.Tests Class Library</Description>
     <AssemblyName>Cdn.Tests</AssemblyName>
-    <VersionPrefix>2.0.0</VersionPrefix>    
+    <VersionPrefix>3.0.0</VersionPrefix>    
```

Comment:
@prakharsharma10 you are not publishing your test projects, so changing version in test projects will not make any difference. I think you should not change this file because this unnecessary adds file history when it's not needed